### PR TITLE
Implement freeze/frost protection support

### PR DESCRIPTION
**Pull request by mill1000**
Command bits are according to dudanov/MideaUART  and reneklootwijk/midea-uart

### CODE REVIEW

File: msmart/device/AC/appliance.py [hunk 1]
```diff
@@ -182,7 +188,7 @@ def update(self, res: state_response):
 
         self._eco_mode = res.eco_mode
         self._turbo_mode = res.turbo_mode
-        self._fahrenheit_unit = res.fahrenheit
+        self._freeze_protection_mode = res.freeze_protection_mode
```
Comment (mill1000) [inline msmart/device/AC/appliance.py:191]: This is a bad change.

**Comment by flyoffacliff** [comment]
Can't wait for this! Is this branch still being developed? :)

**Comment by mill1000** [comment]
My units are put away for the season so I won't be doing much development, but my [future branch](https://github.com/mill1000/midea-msmart/tree/future) has this (and other features) implemented.

**Comment by flyoffacliff** [comment]
> 

I can't figure out how to add your branch to home assistant. What is the difference between the "msmart" repos and the "midea-ac-py" ones? Do we need to install them both?

**Comment by mill1000** [comment]
HACS -> Integrations -> Three dot menu -> Custom repositories

`midea-ac-py` is the HA integration. It provides the interface you see. `midea-msmart` is the underlying library to enable communication. `midea-msmart` is automatically installed when `midea-ac-py` is installed. Although sometimes the wrong version is installed.

**Comment by flyoffacliff** [comment]
Thanks, that makes sense! I think I did end up with the wrong version when I tried to do it manually.

**Comment by mill1000** [comment]
Closing in favor of developing my own fork: [mill1000/midea-msmart](https://github.com/mill1000/midea-msmart)

**Pull request closed without merging**